housingbayarea/bloom
# fix: add signup link to smc

This PR addresses #630 


## Description

This adds the notifications signup section to the SMC site by setting the jurisdiction's `notifications_sign_up_url` variable. 
![image](https://github.com/housingbayarea/bloom/assets/42942267/8afde2ea-fbfa-449f-ae5d-2848005894ef)


## How Can This Be Tested/Reviewed?

After pulling this down locally you can reseed and the `notifications_sign_up_url` value in the DB should be set to the correct URL. Afterwards you should see the new section on the homepage if you set the jurisdiction to "San Mateo"

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
